<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="widgetWindow">
      <item>
       <layout class="QVBoxLayout" name="welcomeScreen">
        <item>
         <widget class="QLabel" name="label_welcomeScreen1">
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Lara</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_welcomeScreen3">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">margin-right:25%;
margin-left:25%;
color:black;</string>
          </property>
          <property name="text">
           <string>Easily convert geographic files to PNG images</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_welcomeScreen2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Choose the file format in the menu above</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_Conversion">
    <property name="title">
     <string>File Format</string>
    </property>
    <addaction name="actionGeoTiff"/>
    <addaction name="actionCSV"/>
    <addaction name="actionGeoJson"/>
    <addaction name="actionGeoPackage"/>
   </widget>
   <widget class="QMenu" name="menuLegacy">
    <property name="title">
     <string>Legacy</string>
    </property>
    <addaction name="actionGeoTiff_Legacy"/>
    <addaction name="actionCSV_Legacy"/>
    <addaction name="actionGeoJson_Legacy"/>
    <addaction name="actionGeoPackage_Legacy"/>
   </widget>
   <addaction name="menuFile_Conversion"/>
   <addaction name="menuLegacy"/>
  </widget>
  <action name="actionGeoTiff">
   <property name="text">
    <string>GeoTiff</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionGeoJson">
   <property name="text">
    <string>GeoJson</string>
   </property>
  </action>
  <action name="actionGeoPackage">
   <property name="text">
    <string>GeoPackage</string>
   </property>
  </action>
  <action name="actionGeoTiff_Legacy">
   <property name="text">
    <string>GeoTiff (Legacy)</string>
   </property>
  </action>
  <action name="actionCSV_Legacy">
   <property name="text">
    <string>CSV (Legacy)</string>
   </property>
  </action>
  <action name="actionGeoJson_Legacy">
   <property name="text">
    <string>GeoJson (Legacy)</string>
   </property>
  </action>
  <action name="actionGeoPackage_Legacy">
   <property name="text">
    <string>GeoPackage (Legacy)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
